packages/strapi-design-system/src/Avatar/Avatar.js

```diff
@@ -62,33 +62,22 @@ export const Avatar = ({ src, alt, preview }) => {
   );
 };
 
-const getColor =
-  (colorName) =>
-  ({ theme }) =>
-    theme.colors[colorName];
-
 const InitialsWrapper = styled(Flex)`
-  width: ${avatarSize}px;
-  height: ${avatarSize}px;
-  border-radius: 50%;
-
-  background: linear-gradient(
-    157deg,
-    ${getColor('primary600')} 0%,
-    ${getColor('primary500')} 50%,
-    ${getColor('primary200')} 81%,
-    ${getColor('primary100')} 96%
-  );
-
   span {
     line-height: 0;
   }
 `;
 
 export const Initials = ({ children }) => {
   return (
-    <InitialsWrapper justifyContent="center">
-      <Typography fontWeight="bold" textColor="neutral0" fontSize={0} textTransform="uppercase">
+    <InitialsWrapper
+      borderRadius="50%"
+      width={`${avatarSize}px`}
+      height={`${avatarSize}px`}
```

Comment:
I'm wondering if we should turn this sizes into rem units so it can scale

Reply:
wdyt?

Reply:
sounds good indeed, on it!

Reply:
mh for some of them it seems to break things

if I do 
```
<AvatarImg src={src} alt={alt} width={`${avatarSize / 16}rem`} height={`${avatarSize / 16}rem`} />
``` 
it gives me this (yes the little dot it the AvatarImg) 😅
<img width="178" alt="image" src="https://user-images.githubusercontent.com/71838159/162438989-9771b658-5fa3-4826-9370-0b285ae7e18a.png">

and for this one
```
        <PreviewContainer
          aria-hidden
          alt=""
          width={`${previewSize / 16}rem`}
          height={`${previewSize / 16}rem`}
          src={preview === true ? src : preview}
        />
```
I get (little dot is the preview:
<img width="92" alt="image" src="https://user-images.githubusercontent.com/71838159/162439264-bf7e2bfb-ee4d-4c73-89f7-12bf764e6ac1.png">

Reply:
I can update AvatarImgWrapper and the Overlay without breaking things though

Reply:
Wut? 😂  you can rollback to px if you can't fix it for the moment 👍🏽